sql/core/src/main/scala/org/apache/spark/sql/api/python/PythonSQLUtils.scala

```diff
@@ -42,6 +42,8 @@ private[sql] object PythonSQLUtils {
 
   def listSQLConfigs(): Array[(String, String, String, String)] = {
     val conf = new SQLConf()
+    // Force to build StaticSQLConf, which is a little bit hacky here
+    conf.warehousePath
```

Comment:
Why don't you just reference `StaticSQLConf` directly here instead of `conf.warehousePath`?

Reply:
Also, I think we should separately document these static SQL configurations instead of mixing with runtime configurations.

Reply:
+1 on the idea to separately document them. Putting them in the same table looks confusing...

Reply:
OK, I agree. I didn't separate them because the command `set -v` doesn't.